scripts/uberenv/vcpkg_ports/uberenv-axom/portfile.cmake

```diff
@@ -0,0 +1,147 @@
+if(VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")
+    message(FATAL_ERROR "${PORT} does not currently support UWP")
+endif()
+
+include(vcpkg_common_functions)
+
+message(STATUS "Building dependencies for Axom")
+
+message(STATUS "CURRENT_INSTALLED_DIR -- ${CURRENT_INSTALLED_DIR}")
+message(STATUS "PORT -- ${PORT}")
+
+set(_copyright [=[
+Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+other Axom Project Developers. See the top-level COPYRIGHT file for details.
+
+SPDX-License-Identifier: (BSD-3-Clause)
+]=])
+
+set(_host-config_hdr [=[
```

Comment:
Please add a comment to indicate that the ``_host-config_hdf`` variable will be used to emit a host-config file.